docs/hub/spaces-config-reference.md

```diff
@@ -2,41 +2,41 @@
 
 Spaces are configured through the `YAML` block at the top of the **README.md** file at the root of the repository. All the accepted parameters are listed below.
 
-**`title`** : _string_
-Display title for the Space.
+**`title`** : _string_  
+Display title for the Space.  
```

Comment:
Not sure what happens here.
when adding spaces at the end of lines, they'll be rendered as a newline? Maybe it would be more explicit to add a `<br>` or just to add an extra newline between the two lines no?

Reply:
See https://daringfireball.net/projects/markdown/syntax#p

As you like but I think that a `<br>` tag will be a bit noisy and make the Markdown / source code version of the doc less readable

Adding an extra newline would create a full paragraph and it's not the intent here

Additional spaces at the end of the blocks have been added so future diffs are cleaner (just like trailing commas)

Reply:
Your solution works for me, but you should add a `<!-- comment -->` for other team members not to delete the trailing whitespace =)